main.cpp

```diff
@@ -465,33 +487,39 @@ int main()
     crushIt.setPersonalTrainer("chuck", 2.5f, 1000);
     Gym::Member example;
     crushIt.sellMonthlyPass(example, 29.99, 6);
-    std::cout << "average user number is: " << crushIt.getUserFreq(45, 30) << "\n";
+    std::cout << "average user number is: " << crushIt.getUserFreq(45, 30) << ", today there were: " << crushIt.mNumDailyPeople << "\n";
+    crushIt.printUserFreq();
     crushIt.checkIncome(34, 30, 24.99);
     
     Gym::Member patron;
+    std::cout << "you have " << patron.passesLeft << " passes left!\n";
     patron.punchPass(5);
     patron.bringAFriend(true, 4);
     patron.giveReward(1, "10% Discount", 10);
     patron.trackProgress(2, 10);
     
     Piano grandPiano;
     grandPiano.volumeDown(false, 1);
+    std::cout << "now playing note " << grandPiano.numKey * grandPiano.playTime << "\n";
     grandPiano.playNote(40, 42.25);
     grandPiano.muteNote(false, false);
-    grandPiano.arpeggio(40, 24, 3);
+    grandPiano.arpeggio(40, 4, 1);
     
     WaveGenerator waveGen;
+    std::cout << "sine wave frequency is: " << waveGen.mSineWaveFreq * waveGen.mOscillator << "\n";
     waveGen.genSineWave(250, 3);
     waveGen.waveMod(4, "Ring");
```

Comment:
`waveMod` is not a verb.
choose function names that describe actions
- [ ] revise